### ISSUE TITLE
[Request] Use System time instead of World time for Calculation of arrival for ships in FTL

### ISSUE THREAD

**Issue by draksterau** [zmaster587/AdvancedRocketry#1845]
**What do you think is missing or what do you think can be done better**
At present you're using the total World Time for the space dimension to calculate 'arrival time' for ships.
I'd recommend you use System time instead, so the space dimension can remain unloaded while no players are online and the next time people log into their ship it will 'instantly drop out of FTL'.

**Comment by draksterau** [comment]
Side note - this may not be relavent for the release version of AR as you're making the max travel time very short by having the fuel cost max out at 500 instead of be calculated on distance.

Still is helpful for future proofing.

**Comment by Gliese-832-c** [comment]
I agree. I already requested much longer travel times, and I'm sure that one day, that will be added, and when this day comes, this config will be very useful.

**Comment by draksterau** [comment]
@Gliese-832-c You *could* try our build - no promises though. 
Check our server out if you'd like to see it in action - stargatemc.com. 
Client and server use the same version.

It has more realistic travel time, been modified to support more than 10k dims as well as a modified holo planet selector.

The only thing its buggy on at the moment is the regular planet selector.

**Comment by dmodoomsirius** [comment]
Feel free to do a pull request. We do accept them. Z reviews it then if he is ok with it merges it in. Otherwise he will nicely request some changes.
